frontend/src/component/commandBar/CommandBar.tsx

```diff
@@ -148,10 +152,25 @@ export const CommandBar = () => {
             link: page.path,
         }));
         setSearchedPages(mappedPages);
+
+        const noResultsFound =
+            query.length !== 0 &&
+            mappedProjects.length === 0 &&
+            mappedPages.length === 0 &&
+            searchedFlagCount === 0;
+        if (noResultsFound) {
+            trackEvent('command-bar', {
+                props: {
+                    eventType: 'no search results found',
+                    query: query,
+                },
+            });
+        }
     }, 200);
 
     const onSearchChange = (value: string) => {
         debouncedSetSearchState(value);
+        setValue(value);
```

Comment:
Brought back the value, because this was actually needed for the UI to be fluid.